README.md

```diff
@@ -255,11 +258,31 @@ topology.metrics.consumer.register:
   - class: "com.verisign.storm.metrics.GraphiteMetricsConsumer"
     parallelism.hint: 1
     argument:
+      metrics.reporter.name: "com.verisign.storm.metrics.reporters.GraphiteReporter"
       metrics.graphite.host: "graphite.example.com"
       metrics.graphite.port: "2003"
       metrics.graphite.prefix: "storm.test"
       metrics.graphite.min-connect-attempt-interval-secs: "5"
 ```
+##### Reporting Metrics to Kafka
```

Comment:
We should mention here (Kafka section) that currently you can report metrics to either Graphite or Kafka, but not to both at the same time.